src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title>EveryBoard 25.1800-4.0</title>
+        <title>EveryBoard 25.1797-4.0</title>
```

Comment:
up to date ?

Reply:
yep, moins de tests car des branches sont devenues impossibles, donc supprimées !